app-modules/engagement/src/Actions/GenerateTipTapBodyJson.php

```diff
@@ -53,7 +53,7 @@ public function __invoke(string $body, array $mergeTags = []): array
                 foreach ($node->content as $item) {
                     preg_match_all('/{{2}[\s\S]*?}{2}|\s*\S+\s*/', $item->text, $tokens);
 
-                    if (empty($tokens)) {
+                    if (! $tokens && $tokens == null) {
```

Comment:
What is this change for?

Reply:
Hi @Orrison 

On previous if condition Lint error is throwing : 

` Variable $tokens in empty() always exists and is not falsy.
` 
 Hence I need to change it from 
 
` if (empty($tokens)) `
 
 to
 
` if (! $tokens && $tokens == null) `

Reply:
This line reads weird to me, is it not the same as `if (blank($tokens)) {` which will check for an empty array or `null`?